lib/telephone_appointments/summary_document_activity.rb

```diff
@@ -0,0 +1,24 @@
+module TelephoneAppointments
+  class SummaryDocumentActivity
+    API_VERSION = 'v1'.freeze
+    PATH = 'summary_documents'.freeze
+
+    def initialize(appointment_id:, guider_uid:)
+      @appointment_id = appointment_id
+      @guider_uid = guider_uid
+    end
+
+    def save
+      @response = TelephoneAppointments.api.post(
+        "/api/#{API_VERSION}/#{PATH}",
+        appointment_id: @appointment_id,
+        guider_uid: @guider_uid
+      )
+      @response.success?
+    end
+
+    def errors
+      @response.errors
```

Comment:
What happens if I call this method without calling `save` first? We should guard against that for a consistent API.